docs/operation/operation.md

```diff
@@ -761,3 +761,87 @@ basis with the rfcPath() filter. See
 If the second interpretation gets considered the right way, and the
 other one a bug, then the default value for this flag may become to
 be on.
+
+## Debugging Requests
+
+Skipper provides [filters](../reference/filters.md), that can change
```

Comment:
I would mention that sometimes the most important information that it shows which request is matched.

Reply:
I changed the text to explain what it shows